website/identifiers/utils.py

```diff
@@ -94,19 +94,24 @@ def request_identifiers_from_ezid(target_object):
 
         client = get_ezid_client()
         already_exists = False
+        new = True
```

Comment:
It seems as though `new` is just the opposite of `already_exists` -- since `already_exists` is being set and used already, can you just use this variable instead of setting and keeping track of a new one?

Reply:
this 'new' is not just opposite of already_exist, 
already_exist applies to any projects that DOI is already generated, which means that project can have valid ARK
new only applies to any project that has DOI after this change, which means that project only have DOI and no ARK.